packages/api/src/controllers/stream.ts

```diff
@@ -696,27 +725,8 @@ app.post(
       return res.json({ errors: ["not found"] });
     }
 
+    const recordingSessionId = req.query.sessionId?.toString();
```

Comment:
WDYT of just calling this `sessionId` everywhere? (also creating the session object with this as the ID, if it's an UUID in the same format as ours)

Reply:
I think we should name it `sessionId` only if it's used as `id` of the `session` entity. We could actually use it as as `session.id` (and child `stream.id`), but I'm not sure we can do it, because I see this comment in the code:

```javascript
    // The first four letters of our playback id are the shard key.
    const id = stream.playbackId.slice(0, 4) + uuid().slice(4);
```

Then, I guess we cannot we want to keep sharding with the playback id. Wdyt?

Reply:
Hm yeah, we have that logic for sharding. It is only really necessary for the actual "transcoding sessions" (child streams) though [1], but have no real use on the sessions! So we must keep the sharding logic for the child streams, but are free to update the sessions as we want! I'd say they should have the `session.id = sessionId` and probably not even have a `playbackId`.

> (and child stream.id)

Note that we cannot use the `sessionId` for the child stream since there can be multiple child streams for the same `sessionId`. The child stream should stay almost the same with this new logic, not changing anything apart from the new field that points to the session ID.

[1] We have special routing configs for the `/live/{sessionId}` API, which routes streams that start with a specific character to a fixed broadcaster instance. I believe this was made to optimize resources on broadcasters etc, which is still valid until we move to catalyst-local broadcasters.